index.js

```diff
@@ -554,16 +548,181 @@
   //.
   //. ```javascript
   //. > S.equals(0, -0)
-  //. false
+  //. true
   //.
   //. > S.equals(NaN, NaN)
   //. true
   //.
   //. > S.equals(S.Just([1, 2, 3]), S.Just([1, 2, 3]))
   //. true
+  //.
+  //. > S.equals(S.Just([1, 2, 3]), S.Just([1, 2, 4]))
+  //. false
   //. ```
   S.equals = def('equals', {a: [Z.Setoid]}, [a, a, $.Boolean], Z.equals);
 
+  //# lt :: Ord a => a -> a -> Boolean
+  //.
+  //. Flipped version of [`lt_`](#lt_) suitable for partial application.
+  //.
+  //. ```javascript
+  //. > S.filter(S.lt(3), [1, 2, 3, 4, 5])
+  //. [1, 2]
+  //. ```
+  S.lt = def('lt', {a: [Z.Ord]}, [a, a, $.Boolean], Z.gt);
+
+  //# lt_ :: Ord a => a -> a -> Boolean
+  //.
+  //. Curried version of [`Z.lt`][].
+  //.
+  //. See also [`lt`](#lt).
+  //.
+  //. ```javascript
+  //. > S.lt_([1, 2, 3], [1, 2, 3])
+  //. false
+  //.
+  //. > S.lt_([1, 2, 3], [1, 2, 4])
+  //. true
+  //.
+  //. > S.lt_([1, 2, 3], [1, 2])
+  //. false
+  //. ```
+  S.lt_ = def('lt_', {a: [Z.Ord]}, [a, a, $.Boolean], Z.lt);
+
+  //# lte :: Ord a => a -> a -> Boolean
+  //.
+  //. Flipped version of [`lte_`](#lte_) suitable for partial application.
+  //.
+  //. ```javascript
+  //. > S.filter(S.lte(3), [1, 2, 3, 4, 5])
+  //. [1, 2, 3]
+  //. ```
+  S.lte = def('lte', {a: [Z.Ord]}, [a, a, $.Boolean], Z.gte);
+
+  //# lte_ :: Ord a => a -> a -> Boolean
+  //.
+  //. Curried version of [`Z.lte`][].
+  //.
+  //. See also [`lte`](#lte).
+  //.
+  //. ```javascript
+  //. > S.lte_([1, 2, 3], [1, 2, 3])
+  //. true
+  //.
+  //. > S.lte_([1, 2, 3], [1, 2, 4])
+  //. true
+  //.
+  //. > S.lte_([1, 2, 3], [1, 2])
+  //. false
+  //. ```
+  S.lte_ = def('lte_', {a: [Z.Ord]}, [a, a, $.Boolean], Z.lte);
```

Comment:
sanctuary-js/sanctuary-def#147 provides a way to prevent `S.lte` and friends from being applied to two arguments at once. Preventing `S.lte_` and friends from being partially applied is not so easy.